### PR TITLE
Fixing "Observe can be null" bug

### LINKED ISSUE #6644: this.visibilityObservable_ can be null
Missing null check [here](https://github.com/ampproject/amphtml/blob/43f624171de67d305e86197d51ef30e3a6f1ae9c/src/service/document-state.js#L120).

In fact, there might be other places missing null check due to PR #6496

### PR DESCRIPTION
**Pull request by chenshay**
close #6644

### CODE REVIEW

File: src/input.js [hunk 1]
```diff
@@ -192,6 +192,9 @@ export class Input {
     }
 
     this.keyboardActive_ = true;
+    if (!this.keyboardStateObservable_) {
+      this.keyboardStateObservable_ = new Observable();
```
Comment (jridgewell) [inline src/input.js]: No need to create these, we can just skip the `#fire` instead.

**Comment by lannka** [comment]
please also address Malte's comments in the preview PR.
Or you prefer to do in a separate PR?

**Comment by cramforce** [comment]
I'd prefer if we reverted the PR and addressed all comments in rollforward.

**Comment by chenshay** [comment]
@lannka done.

**Pull request closed without merging**